src/nnet3/nnet-diagnostics.h

```diff
@@ -97,6 +97,11 @@ class NnetComputeProb {
   // or NULL if there is no such info.
   const SimpleObjectiveInfo *GetObjective(const std::string &output_name) const;
 
+  // return objective info for all outputs
```

Comment:
I think it would be nicer to return a string that would summarize the objectives for different outputs, in a way that could be used more easily by the calling code (and would look nice in the normal case where there was just one output).